easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.70.0-gompic-2019a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'EB_Boost'
+
+name = 'Boost.Python'
+version = '1.70.0'
+
+homepage = 'http://boostorg.github.io/python'
+description = """Boost.Python is a C++ library which enables seamless interoperability between C++
+ and the Python programming language."""
+
+toolchain = {'name': 'gompic', 'version': '2019a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://download.sourceforge.net/boost/']
+sources = ['boost_%s.tar.gz' % '_'.join(version.split('.'))]
+patches = ['Boost-%(version)s_fix-Python3.patch']
+checksums = [
+    '882b48708d211a5f48e60b0124cf5863c1534cd544ecd0664bb534a4b5d506e9',  # boost_1_70_0.tar.gz
+    '48d1379df61e8289c5f98ff024a2b0429d49a7e06d7d88186d44a0a0254c4347',  # Boost-1.70.0_fix-Python3.patch
+]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [('Boost', version)]
```

Comment:
doesn't this require a `Boost-1.70.0-gompic-2019a.eb` easyconfig?

Reply:
Yes, and that's there (otherwise the tests would fail)

Reply:
Hmm, seems like it's no longer there in `develop`...

Apparently it got (incorrectly) archived in #8557, so I'll restore it.